Fido.ps1

```diff
@@ -801,7 +801,7 @@ function Get-Windows-Download-Links([int]$SelectedVersion, [int]$SelectedRelease
 
 		try {
 			$Is64 = [Environment]::Is64BitOperatingSystem
-			$r = Invoke-WebRequest -Method Post -UseBasicParsing -UserAgent $UserAgent -WebSession $Session $url
+			$r = Invoke-WebRequest -Method Post -Headers @{'Referer' = 'https://www.microsoft.com/software-download/windows11'} -UseBasicParsing -UserAgent $UserAgent -WebSession $Session $url
```

Comment:
I don't think we want to do that for now, because `https://www.microsoft.com/software-download/windows8.1` is not a valid page, and that's the URL we'd get for 8.1...
As long as Microsoft's server don't seem to care about the version, using window11 should be good enough.